frontend/src/pages/account-settings/components/subscription/styles.module.scss

```diff
@@ -52,7 +52,7 @@
 
 .subscription-buttons {
   display: grid;
-  grid-template-columns: 1fr 1fr;
+  grid-template-columns: 1fr;
```

Comment:
Need to add button size. For example, "Change Password" modal

Reply:
I agree, but I'm not sure that it's worth changing, as far as I understand, there should be another button in the future, that's why I didn't change the styles

Reply:
And you will need to pass an additional prop from stripeCheckout , because this is not a button and it does not have a className